cmds/graph.c

```diff
@@ -279,7 +279,8 @@ static struct task_graph *get_task_graph(struct uftrace_task_reader *task, uint6
 	graph = get_graph(task, time, addr);
 
 	if (tg->utg.graph && tg->utg.graph != graph) {
-		pr_dbg("detect new session: %.*s\n", SESSION_ID_LEN, graph->sess->sid);
+		if (graph && graph->sess)
```

Comment:
I'm not sure this can happen actually.  Basically graphs are created for existing sessions.  You can see a NULL `graph` but never see a NULL `graph->sess`, right?